### PR TITLE
feat(types): add form field type definitions

### PR DESCRIPTION
**Pull request by csabageiszt**
__Use case__
Importing Svelte Materialify form field components without type definitions in a TypeScript Svelte project with `strict` compiler options breaks the production build.

Relates and partial solution to #78. 

__Changes__
* Add type definitions for `Checkbox`, `Radio`, `Select`, `Slider`, `Switch`, and `Textarea`. 
* Change single line comments to block comments and add missing props in `TextField` definition.

### CODE REVIEW

File: packages/svelte-materialify/@types/TextField.d.ts [hunk 1]
```diff
@@ -1,42 +1,55 @@
...
-  // Creates counter for input length.
+  /** Display a counter set to a desired input length. */
   counter?: number;
-  // An array of functions which take input value as arguement and return error message.
-  rules?: ((value) => string | true)[];
-  // Delays validation till blur.
+  /** Error messages to display. */
+  messages?: string[];
+  /**
+   * A list of validator functions that take the textarea value and return an error
```
Comment (Florian-Schoenherr) [inline packages/svelte-materialify/@types/TextField.d.ts:37]: In this file you have called it `textarea` in some places

File: packages/svelte-materialify/@types/Select.d.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { SvelteComponent } from './shared';
+
+interface SelectProps<T extends number[] | string[] = string[]> {
+  /** Classes to add to select wrapper. */
+  class?: string;
+  /** Whether select is opened. */
+  active?: string;
```
Comment (Florian-Schoenherr) [inline packages/svelte-materialify/@types/Select.d.ts:7]: active is a boolean, I believe

**Review by Florian-Schoenherr — CHANGES_REQUESTED** [review]
This is of huge help! Thank you!
Reviewed, also thank you for sorting index.d.ts :)

File: packages/svelte-materialify/@types/Select.d.ts (hunk 2 above)
Comment (csabageiszt) [reply packages/svelte-materialify/@types/Select.d.ts]: Thanks for the review. I fixed these issues, and some more. Generic prop typing seemed like a good idea, but it does not work in practice, so I removed it.

**Review by Florian-Schoenherr — APPROVED** [review]

**Pull request merged**